### PR TITLE
Sidebar Duotone icons

### LINKED ISSUE #9216: Update side navigation icons with new duotone design
**Is your feature request related to a problem? Please describe.**
No issues with the current icons; this request is for a design enhancement.

**Describe the solution you’d like**
Replace the existing side navigation icons with redesigned duotone icons that align with the platform’s design system.

### PR DESCRIPTION
**Pull request by shivankacker**
> Waiting for facilities icon

## Proposed Changes

- Fixes #9216

@ohcnetwork/care-fe-code-reviewers

## Merge Checklist

- [ ] Add specs that demonstrate bug / test a new feature.
- [ ] Update [product documentation](https://docs.ohc.network).
- [ ] Ensure that UI text is kept in I18n files.
- [ ] Prep screenshot or demo video for changelog entry, and attach it to issue.
- [ ] Request for Peer Reviews
- [ ] Completion of QA

## Release Notes

- **New Features**
	- Introduced duo-tone icons, enhancing the visual representation of icons in the application.
	- Added a new JSON file for duo-tone icon paths, expanding available icon options.

- **Bug Fixes**
	- Updated navigation icons in the sidebar for improved semantic clarity.

- **Improvements**
	- Enhanced notification handling with better state management for read/unread statuses.
	- Improved UI interactions and loading indicators for notifications.
	- Updated various components to use new icon representations, ensuring consistency across the application.

### CODE REVIEW

File: src/CAREUI/icons/CareIcon.tsx [hunk 1]
```diff
@@ -26,11 +27,15 @@ export default function CareIcon({
 }: CareIconProps) {
   // TODO: fill & strokeWidth are defined for only one icon
   // Rethink Implementation
-  const [viewBox, path, fill, strokeWidth] = iconData[icon] as [
+  const [viewBox, path, fill, strokeWidth, secondaryPath] = {
+    ...iconData,
+    ...duoToneIconData,
+  }[icon] as [
```
Comment (sainak) [inline src/CAREUI/icons/CareIcon.tsx]: this seems inefficient, it will create new object per icon per render

File: src/CAREUI/icons/CareIcon.tsx [hunk 2]
```diff
@@ -50,6 +55,15 @@ export default function CareIcon({
...
       />
+      {secondaryPath && (
+        <path
+          d={secondaryPath}
+          fill={fill === false ? "none" : "currentColor"}
+          stroke={fill === false ? "currentColor" : undefined}
+          strokeWidth={fill === false ? strokeWidth : undefined}
+          opacity={0.32}
+        />
+      )}
```
Comment (sainak) [inline src/CAREUI/icons/CareIcon.tsx:67]: we have a few duotone icons but many plain icons, so doing this condition check for all of them might be unnecessary
instead, can we make duotone icons a separate component?

Comment (shivankacker) [reply src/CAREUI/icons/CareIcon.tsx:67]: It is there for future proofing. Duotone icons might depend on these conditions. Creating a separate component would be weird 🤔

Comment (rithviknishad) [reply src/CAREUI/icons/CareIcon.tsx:67]: An svg can have multiple paths right? So why don't we allow an array of paths instead?

Comment (shivankacker) [reply src/CAREUI/icons/CareIcon.tsx:67]: @rithviknishad we would need to change our existing icon schema for that. I am up for it, but thought its best to leave it be for the current scope

**Comment by nihal467** [comment]
LGTM

**Review by rithviknishad — APPROVED** [review]

**Review by khavinshankar — APPROVED** [review]

**Pull request merged**